Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: simpler quitting flow

### Description (*)

This PR implements the same feature as #537 did, but without checking scene names in code. To do this, the SettingsPanelCanvas is removed from the DDOL scene, and added to every scene. An enum field in QuitPanel decides if we want to return to the menu, or quit the application, so each scene can specify the expected behavior. In this case, in every scene except the MainMenu, we return to the menu when using the quit button, and in the MainMenu scene, we quit the application.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2919](https://jira.unity3d.com/browse/MTT-2919)
### Manual testing scenarios

1. Launch the game and use the quit button in the different scenes
2. See that it brings you back to the main menu in all scenes except the MainMenu scene, where it quits the application
### Questions or comments

### Contribution checklist
